### PR TITLE
Fix svm drop columns test

### PR DESCRIPTION
**Pull request by valenad1**
Sorry for another PR, I change the base to rel-zermelo and bad things happened. (https://github.com/h2oai/h2o-3/pull/5112)

We already discus and I implement all suggestions into pyunit_PUBDEV_5705_drop_columns_parser_svmlight_fail.py.

@michalkurka pointed out, the original test also tests the correct behavior.

I created separate test for correct behavior with some improvements. Please focus mainly on pyunit_PUBDEV_5705_drop_columns_parser_svmlight_large.py 

Thanks

### CODE REVIEW

**Review by wendycwong — APPROVED** [review]

File: h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_large.py [hunk 1]
```diff
@@ -8,67 +8,38 @@
 
 
 def test_parser_svmlight_column_skip():
-    # generate a big frame with all datatypes and save it to svmlight
```
Comment (michalkurka) [inline h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_large.py:11]: 10k rows and 10 columns doesn't seem large to me

Should all the references to "big" be removed and test renamed (`large` classifier dropped)? The classifiers are important - they denote at what test group will the test run.

File: h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_fail.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+sys.path.insert(1, "../../")
+import h2o
+from tests import pyunit_utils
+import os
+from h2o.exceptions import H2OResponseError
+
+
+def test_parser_svmlight_column_skip_not_supported():
+    print("Test that functions calling fail if skipped_columns is passed with svm file.")
+    # generate a big frame with all datatypes and save it to svmlight
```
Comment (michalkurka) [inline h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_fail.py:13]: still says big here ;)

File: h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_large.py (hunk 1 above)
Comment (wendycwong) [reply h2o-py/tests/testdir_parser/pyunit_PUBDEV_5705_drop_columns_parser_svmlight_large.py:11]: @michalkurka

Good call.  Yes, the test can now be moved to normal and the test name will be: pyunit_PUBDEV_5705_drop_columns_parser_svmlight.py

**Review by michalkurka — APPROVED** [review]
Thank you!

**Pull request merged**